packages/avatars/src/styled/StyledStandaloneStatusIndicator.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { math } from 'polished';
+
+import { IStatusIndicatorProps, STATUS } from '../types';
+import { getStatusColor, statusIconStyles, TRANSITION_DURATION } from './utility';
+
+const COMPONENT_ID = 'avatars.status-indicator.indicator';
+
+const [available, away, transfers, offline] = STATUS;
+
+const sizeStyles = (props: IStatusIndicatorProps & ThemeProps<DefaultTheme>) => {
+  const borderWidth = props.theme.shadowWidths.sm;
+
+  let size = '0';
+  let marginTop = props.theme.space.base;
+
+  if (props.isCompact) {
+    marginTop = props.theme.space.base * 1.5;
+    size = math(`${props.theme.space.base * 3}px - (${borderWidth} * 2)`);
+  } else {
+    size = math(`${props.theme.space.base * 4}px - (${borderWidth} * 2)`);
+  }
+
+  return css`
+    margin: ${props.theme.space.base}px;
+    margin-top: ${marginTop}px;
+    border: ${borderWidth} ${props.theme.borderStyles.solid};
+    border-radius: ${size};
+    width: ${size};
+    min-width: ${size};
+    max-width: calc(2em + (${borderWidth} * 3));
```

Comment:
I'm concerned that the use of `em` units here is brittle with respect to the system + theming.

Reply:
This was inherited from the initial implementation, and actually serve no purpose here. Will remove.